Documentation/network/clustermesh/clustermesh.rst

```diff
@@ -94,6 +96,15 @@ time of Cilium:
  * Helm options ``cluster.name`` and ``cluster.id``
  * Cilium CLI install options ``--set cluster.name`` and ``--set cluster.id``
 
+Review :ref:`_k8s_quick_install` for more details and use cases.
```

Comment:
Looks like the Netifly build is failing because this link isn't quite right. Can you change it to `k8s_install_quick`?

```suggestion
Review :ref:`k8s_install_quick` for more details and use cases.
```